php8.3/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM php:8.3-apache-bookworm
```

Comment:
this file is a copy of https://github.com/thebiggive/docker-php/blob/b8e9c544addee6716c42a99d6e27368a11638f8c/Dockerfile#L11-L10 , just changed 8.1-buster to 8.3-bookworm